debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1290,7 +1299,11 @@ public Optional<String[]> parseSignallingMessage(Struct value) {
         });
     }
 
-
+    protected static int validateYBHostname(Configuration config, Field field, Field.ValidationOutput problems) {
+        // YB Note: Not validating the hostname against any pattern.
```

Comment:
why not validating? It should be either single or multi hostname pattern?

Reply:
Added validation and tests for validation.